src/fillTank.test.js

```diff
@@ -1,11 +1,188 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it('should be declared', () => {
+    expect(fillTank).toBeInstanceOf(Function);
+  });
+
+  it('should fill tank fully with no amout', () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
+
+    fillTank(customer, 50);
+
+    expect(customer.money).toBe(1400);
+    expect(customer.vehicle.fuelRemains).toBe(40);
+  });
+
+  it('should fill tank fully with amout bigger than tank capacity', () => {
```

Comment:
The test description 'should fill tank fully with amout bigger than tank capacity' contains a typo. It should be 'amount' instead of 'amout'.

Reply:
done